README.md

```diff
@@ -0,0 +1,376 @@
+# Plug and Play samples spec for May 2020 release
+
+> This spec describes the e2e sample we will use for the next BugBash, and eventually will be published as a tutorial in docs.ms.com.
+
+## Intro
+
+PnP is a set of technologies designed to make it easier to build IoT Solutions based on Azure IoT Hub. To address the needs for device and cloud developers the tutorial must target both.
+
+Devices connected to IoT Hub can de described with DTDL, in terms of:
+
+- Properties (Available trough Twins)
+- Commands (Using the Hub Direct Methods feature)
+- Telemetry (processed through EventHub)
+
+### Bug Bash notes
+
+For a BugBash, the instructions should not duplicate content already  available like READMEs or Docs, instead the instructions should help to find information that is not easy to find (not yet indexed, not available publicly, etc.. )
+
+Each section in the Bug Bash can have different activities targeting different levels of complexity:
+
+1. **Basic**. These are the minimal instructions to complete the scenario. This is the _happy path_ and should be easy to complete. Eg. Send temperature
+2. **Intermediate**. Offer additional activities to explore the feature in depth. Eg. Use different semantic data types.
+3. **Advanced**. Additional activities that will stress the feature being evaluated. Eg. Create telemetry using complex object types.
+
+### Private environments, and preview SDKs
+
+As the time of writing the version of IoT Hub supporting PnP BugBash 2, 08 May2020 release is only deployed to a private hub instance:
+
+```
+<Connection Strings for IoT Hub>
+```
+
+The SDKs required to use these features are in the `public-preview-pnp` branches
+
+The tools are available as internal previews:
+
+- Azure IoT Explorer (TBD)
+- VSCode extension for DTDL (TBD)
+
+## Sample Solution  Features
+
+The solution to build includes 2 devices connected to Azure IoT using Central (SaaS) or Hub (PaaS). 
+
+SaaS solution require to configure th Central application creating device templates based on the DTDL models. Central does not supoprt DTLD v2. It's out of scope for this BugBash.
```

Comment:
I'd remove the notes of Central here instead using Node service sample.

Reply:
I think we should mention that Central does not support v2 yet.